qe_pre_installation_tasks.yml

```diff
@@ -10,6 +10,17 @@
       yum:
         name: '*'
         state: latest
+    - name: Install useful tools
+      yum: name={{ item }} state=installed
+      with_items:
+        - bash-completion
+        - bind-utils
+        - git
+        - psmisc
+        - tmux
+        - tree
+        - vim
+      ignore_errors: yes
```

Comment:
Would it make sense to have this in a separate playbooks (something like `qe_tools.yml`) so that I can rerun it separately when needed?